agent/battle/perception/perception.py

```
@@ -0,0 +1,238 @@
+
+
+# OCR 文本 -> CardColor 映射
+_CARD_TEXT_MAP = {
+    "力击": CardColor.BUSTER,
+    "技击": CardColor.ARTS,
+    "迅击": CardColor.QUICK,
+}
+
+
+def _detect_card(context, img, ui_slot: int) -> CommandCard:
+    # OCR 识别卡牌文字（力击/迅击/技击）
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)